org.eclipse.mylyn.releng/oomph/Mylyn.setup

```diff
@@ -176,7 +186,7 @@
     <setupTask
         xsi:type="setup.targlets:TargletTask"
         id="mylyn.target.api"
-        predecessor="git.clone.mylyn //@projects[name='mylyn']/@setupTasks.2 //@projects[name='mylyn']/@setupTasks.3"
```

Comment:
Just an FYI.  If you want to avoid such positional references to other setup tasks, you can give the target task an unique ID is is the case for the clone task already.

Reply:
OK, I will rework the file and make sure that at least all task listed in predecessor have an unique ID.